public/app/features/explore/ContentOutline/ContentOutline.tsx

```diff
@@ -33,9 +33,12 @@ const getStyles = (theme: GrafanaTheme2) => {
 };
 
 export function ContentOutline({ scroller, panelId }: { scroller: HTMLElement | undefined; panelId: string }) {
+  const { outlineItems } = useContentOutlineContext();
   const [expanded, toggleExpanded] = useToggle(false);
+  const [activeItemId, setActiveItemId] = useState<string | undefined>(outlineItems[0]?.id);
   const styles = useStyles2((theme) => getStyles(theme));
-  const { outlineItems } = useContentOutlineContext();
+  const scrollerRef = useRef(scroller as HTMLElement);
```

Comment:
```suggestion
  const scrollerRef = useRef(scroller || null);
```

This should do the trick to remove the assertion and make the betterer happy ☕